### PR TITLE
Removed Telemetry part

### PR DESCRIPTION
**Pull request by joinimran**
In azure, Telemetry option was removed so I have updated the documentation accordingly. 

Problem:  https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4392

### CODE REVIEW

**Comment by jvsam** [comment]
Hi @joinimran kindly see my questions on #4392. If the [OMS portal is no longer accessible](https://techcommunity.microsoft.com/t5/Windows-Analytics-Blog/Goodbye-OMS-hello-Azure/ba-p/324006), should the steps be updated based on the Azure portal experience? Can we also add the relevant Azure links? 

Will appreciate your insight as well @nenonix. Thank you.

**Comment by joinimran** [comment]
@jvsam , I have tested these steps on Azure portal added the required information as it was happening on Azure portal. Steps added in current PR are verified through azure.

File: windows/deployment/update/windows-analytics-FAQ-troubleshooting.md [hunk 1]
```diff
@@ -230,11 +232,9 @@ We have identified an incompatibility between AbnormalShutdownCount and the Limi
 
 If you want to stop using Upgrade Readiness and stop sending diagnostic data to Microsoft, follow these steps:
 
-1. Unsubscribe from the Upgrade Readiness solution in Azure Portal. In Azure Portal, go to **Settings** > **Connected Sources** > **Windows Telemetry** and choose the **Unsubscribe** option.
+1. Delete the Upgrade Readiness solution in Log Analytics Workspace. In Log Analytics Workspace, go to **Solutions > Compatibility Assessment > Delete** 
 
-   ![Upgrade Readiness unsubscribe](images/upgrade-analytics-unsubscribe.png)
-
-2. Disable the Commercial Data Opt-in Key on computers running Windows 7 SP1 or 8.1. On computers running Windows 10, set the diagnostic data level to **Security**:
+   2. Disable the Commercial Data Opt-in Key on computers running Windows 7 SP1 or 8.1. On computers running Windows 10, set the diagnostic data level to **Security**:
```
Comment (illfated) [inline windows/deployment/update/windows-analytics-FAQ-troubleshooting.md]: Please remove the 3x blank space characters before `2. ` to keep the alignment with `1. ` above.

**Comment by joinimran** [comment]
Closing this PR as I have to delete my fork and I can not do any suggested changes in the article. So going to create a new PR for this.

**Comment by joinimran** [comment]
You can use PR https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4687 for further actions.

@jvsam @illfated FYI Please.

**Pull request closed without merging**